src/test.js

```diff
@@ -44,11 +45,18 @@ var resetTimeout = function () {
   }, self.ms);
 };
 
+var absoluteToRelativePath = function(stack) {
+  for (var i = 0; i< stack.length; i++) {
+    stack[i] = stack[i].replace(process.env.GAUGE_PROJECT_ROOT + path.sep, "");
```

Comment:
use (`path.relative`)[https://nodejs.org/api/path.html#path_path_relative_from_to] instead?

Reply:
To use path.relative, we need to extract and parse the path in the stacktrace and then use the function which will add some extra steps. We need to also check if the path we are dealing with is a project file path because if it is an external library file then path.relative will  show the path relative to the project root path.